desc/compute/_profiles.py

```diff
@@ -216,6 +216,26 @@ def _Te_rr(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="<ne>_rho",
+    label="\\bar{n}_e",
+    units="m^{-3}",
+    units_long="1 / cubic meters",
+    description="Line-averaged electron density",
+    dim=0,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="",
+    data=["ne"],
+)
+def _bar_ne(params, transforms, profiles, data, **kwargs):
+    data["<ne>_rho"] = jnp.sum(data["ne"] * transforms["grid"].spacing[:, 0]) / jnp.sum(
+        transforms["grid"].spacing[:, 0]
+    )
```

Comment:
Can you add comment explaining your assumptions on the grid?
If you're just doing a simple average over $\rho$, don't you want to require a tensor product grid and do
```python
# or reshape in fortran order for LinearGrid etc.
data["<ne>_rho"] = (data["ne"] * grid.spacing[:, 0]).reshape(num_rho, num_theta, num_zeta).sum(axis=0).mean() 
```

Reply:
or if $n_e$ is assume to be radial profile then compress and then integrate

Reply:
Usually it will only be a 1D radial profile, but I suppose we should allow for 3D profiles. Is there any way we can handle general grids like `ConcentricGrid`? If not, how do we specify in the data index that it has to be a tensor product grid?

Reply:
If you have a 3d profile, I think what you want to do is like

```python
data["<ne>_rho"] = np.dot(
  surface_averages(grid, data["ne"], data["|e_theta x e_zeta|"], expand_out=False),
  grid.compress(grid.spacing[:, 0])
)
```
my choice of `|e_theta x e_zeta|` as the Jacobian as opposed to `sqrt(g)` is based off how we compute `data["A"]` as 
```python
# Simple toroidal average A₀ = ∫ A(ζ) dζ / (2π) matches the convention for the
# average major radius R₀ = ∫ R(ρ=0) dζ / (2π)
```
There, when we compute $A(z)$, we use the 2d jacobian `|e_rho x e_theta|` and not `sqrt(g)`.

Reply:
Thanks, I updated to your suggestion

Reply:
including `|e_rho x e_theta|` means this has units of 1/m not 1/m^3.

Usually the line average is something like $\int f(x) dx / \int dx$ where $x$ is a real space coordinate like R or Z. I think the least bad approximation of that in flux coordinates would be something like `dx ~ <sqrt(g^rr)>_fsa dr`

Reply:
It's a surface average not an integral, so the units do not change

Reply:
just do vol avg of ne